examples/1-getting-started/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start:server": "node ./src/server/index.js",
     "start:client": "sleep 1 && yarn scaffold && parcel serve -p 3000 public/index.html",
-    "scaffold": "../../generator/mst-gql-scaffold.js --roots 'Todo' --excludes 'CacheControlScope,Query,Subscription' --outDir src/app/models/ --format ts http://localhost:3001/graphql",
+    "scaffold": "node ../../generator/mst-gql-scaffold.js --roots Todo --outDir src/app/models/ --format ts http://localhost:3001/graphql",
```

Comment:
This file `mst-gql-scaffold` contains `#!/usr/bin/env node` so there is no need to execute it with node

Reply:
I need the `node ` bit or the script doesn't pick up any of the command-line arguments and tries to run with all the defaults.. If you don't see the same thing happening, the difference might be that I'm running Windows 10? Are you running linux/macos? And does the package script still run as expected *with* the `node ` bit?

Reply:
I'm on both mac and linux and it works fine there so this seems to be an issue with windows then. Which version of node are you on?

Reply:
I'm on node v10.16.0 & yarn v1.16.0. I'll see if upgrading fixes it for me.

Reply:
I'm on node v10.1.6.0 and yarn v1.16.0. I'll try upgrading to see if that fixes it for me.

Reply:
@mattiasewers After upgrading to the latest node LTS (v10.16.3) and latest yarn (v1.17.3) I'm still experiencing the same problem. Does my change break things for you? Or can we keep it for improved portability?

Reply:
If this is needed to run things on windows and does not break other platforms I feel we should leave it in